### DISCUSSION TITLE
March Copilot Community Check-in

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#113541] [Copilot News and Announcements]
> “We can only see a short distance ahead, but we can see plenty there that needs to be done” - Alan Turing

👋 Hello everyone! Welcome to our latest Copilot check-in session. As we march through March, I trust you're all marching towards your goals – just as we are! This past month we made some big Copilot announcements and made key changes. 

🔢 Let’s tune in and go!

## ✨ Community Champions: Shoutout 📢  to these members

- Let's start with giving a round of applause 👏 to @iamroberthanson for his outstanding contributions in our forum! 🌟 His assistance in resolving the [Copilot issue with Intellij IDEA](https://github.com/orgs/community/discussions/110488) helped many other users.
  - A quick update: we've re-enabled the version 1.2.9 for now so that IntelliJ 2021 users can continue using Copilot. We do, however, recommend staying up to date as we can’t  commit to supporting older versions.
- Hats off to @DeandrePetersen! 🎉 Their solution to the intricate [certificate validation issue with WSL](https://github.com/orgs/community/discussions/112148) is commendable. For any network-related challenges with Copilot, our Docs offer comprehensive [troubleshooting guide](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-network-errors-for-github-copilot#troubleshooting-certificate-related-errors). Keep up the fantastic work! 👍
- A special shoutout to @PhilipBright for his self-solve! Thanks for updating us on how other [extensions can impact Copilot initialization](https://github.com/orgs/community/discussions/109822#discussioncomment-8575955). Sometimes, a simple cleanup of unused extensions can work wonders. Keep sharing your wisdom with the community!"

This month, we've seen some great contributions from members of our community. We've acknowledged these contributions, and now it's your turn to show your appreciation. When your question gets answered, simply click the Mark as Answer button. This small action can make a big difference for others facing similar queries. ✅

### 💼  [GitHub Copilot Enterprise is now generally available](https://github.blog/changelog/2024-02-27-copilot-enterprise-is-now-generally-available)

Copilot Enterprise, our most advanced AI offering to date, was made generally available last month. With Copilot Enterprise, you can:
- ⚡ Gain a deeper understanding of your organization’s unique codebase
- ✨ Quickly access organizational knowledge and best practices
- ⏩ Review pull requests faster 

[Learn more about GitHub Copilot Enterprise ](https://github.com/orgs/community/discussions/110324)

### 🚢  [GitHub Copilot Chat General Availability in JetBrains IDE](https://github.blog/changelog/2024-03-07-github-copilot-chat-general-availability-in-jetbrains-ide)

Copilot Chat is now officially available in JetBrains IDEs for all Copilot Individual, Business, and Enterprise customers. Have questions, feedback, or want to learn more? [Join the discussion](https://github.com/orgs/community/discussions/110614)!

### [⏫ Copilot Code Completion model updated with more improvements](https://github.blog/changelog/2024-02-27-copilot-february-27th-update/#copilot-code-completion-model-updated-with-more-improvements)

New update for the Copilot Code Completion model across all IDEs! This update enhances instruction following and boosts performance, ensuring Copilot generates code that aligns more closely with user intent and requirements. Additionally, users can expect improved speed and efficiency for certain tasks.

## Explore the Learniverse 🔖 

- [**GitHub Copilot Learning 🧑‍🎓  Pathway**](https://github.blog/2024-03-04-github-copilot-learning-pathway-accelerate-your-business-with-ai/) - Learn what GitHub Copilot can help your business achieve in this expert-guided GitHub Learning Pathway, featuring insights from tech leaders at top organizations.
- [**Hard and soft skills for developers 👨‍💻  coding in the age of AI**](https://github.blog/2024-03-07-hard-and-soft-skills-for-developers-coding-in-the-age-of-ai/) - While AI revolutionizes software development, it still relies on developers to pilot its use. This blog covers the skills that developers need to have for navigating this new AI-powered coding frontier.
- [**Customizing and fine-tuning 🤖  LLMs: What you need to know**](https://github.blog/2024-02-28-customizing-and-fine-tuning-llms-what-you-need-to-know/) - Learn how your organization can customize its LLM-based solution through retrieval augmented generation and fine-tuning.
- **[Accelerate ⚡  test-driven development with GitHub Copilot](https://github.com/readme/guides/github-copilot-automattic)** - Discover 🌐  practical insights in this guide on using GitHub Copilot for Swift development. Learn to streamline Test-driven development feedback loops, reshape code efficiently, and generate complete test doubles with ease.
- **[How we’re experimenting with LLMs to evolve GitHub Copilot 😮 ](https://github.blog/2023-12-06-how-were-experimenting-with-llms-to-evolve-github-copilot/)** - Learn how we’re experimenting with generative AI models to extend GitHub Copilot across the developer lifecycle.
- **[How to build 🏗️  an enterprise LLM application: Lessons from GitHub Copilot](https://github.blog/2023-09-06-how-to-build-an-enterprise-llm-application-lessons-from-github-copilot/)** - What’s better than hearing from the OGs. The team behind GitHub Copilot shares its lessons for building an LLM app that delivers value to both individuals and enterprise users at scale.

That’s it for March. Thank you for your active participation and valuable contributions. Take a moment to share what you've achieved with Copilot in the recent past ❓ , as your experiences can inspire and guide others on their journey.

**Comment by fisforfaheem** [comment]
Kindly add FULL chantgpt 4 turbo support

**Comment by Ry-DS** [comment]
I'm hoping to bring more attention to this issue since it's been bothering a lot of us as of late. Thank you! 
https://github.com/orgs/community/discussions/72934
